src/stability_sdk/animation_ui.py

```diff
@@ -0,0 +1,601 @@
+import glob
+import gradio as gr
+import json
+import locale
+import os
+import param
+import subprocess
+
+from collections import OrderedDict
+from tqdm import tqdm
+from typing import Any, Dict, List, Optional
+
+from .api import (
+    ClassifierException, 
+    Context,
+    Project,
+    OutOfCreditsException,
+)
+from .animation import (
+    AnimationArgs,
+    Animator,
+    AnimationSettings,
+    BasicSettings,
+    CameraSettings,
+    CoherenceSettings,
+    ColorSettings,
+    DepthSettings,
+    InpaintingSettings,
+    Rendering3dSettings,
+    VideoInputSettings,
+    VideoOutputSettings,
+)
+
+DATA_VERSION = "0.1"
+DATA_GENERATOR = "alpha-test-notebook"
+
+PRESETS = {
```

Comment:
This is good for now. It may be good to have these specified in a .json file (or files). Without use of asset service, it makes sense we haven't yet moved in that direction.